pycqed/measurement/quantum_experiment.py

```diff
@@ -501,6 +510,9 @@ def _configure_mc(self, MC=None):
                                 "free to make a pull request ;). Skipping compression"
                                 "for now.")
 
+        if self.harmonize_element_lengths:
+            self.sequences[0].harmonize_element_lengths(self.sequences)
```

Comment:
This technique is also used at other points in pycqed, e.g. for merging sequences. The point is that the staticmethod is available wherever the sequence object is available, without the need to explicitly include the module. Moreover, we could in the future decide to have special kinds of sequences which bring their own overridden staticmethods if needed. In this case, calling the staticmethod via the instance automatically gives the right version of the staticmethod. If you have strong arguments against this way of implementing it, we should discuss it also for other points in pycqed where the same is done. Otherwise, I would just keep it as it is now.

Reply:
I still think that the current code is confusing to read, as it assigns a special role to the first sequence `sequences[0]`.
Imports are handeled automatically by most IDEs, so I don't really see the large downside of having an extra import for the class.
We could look at the other instances where this is done as well.